cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -67,7 +67,7 @@
                 {% if aria_group_title %}
                 aria-labelledby="{{ aria_group_title | slugify ~ '-menu' }}"
                 {% elif nav_title %}
-                aria-label="{{ nav_title | safe }}"
+                title="{{ nav_title | safe }}"
```

Comment:
Would also be nice to update this to indicate that this is one of multiple lists under the same section.

```suggestion
                title="{{ nav_title | safe }} group {{ loop.index }} of {{ loop.length }}"
```